.tekton/determine-image-digest.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: tekton.dev/v1
```

Comment:
Regarding the filename `determine-image-digest.yaml`: our conventions are made up and not declared anywhere but it still helps to do things consistently even though conventions might not be explicit. If you check `.tekton/`, you'll see other files with `Task`-s have their names ending with `-task.yaml`. I suggest this file should be no exception and be called `determine-image-digest-task.yaml`.

Reply:
done